HotFix.Test/utilities/scheduling/get_active.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using HotFix.Core;
+using HotFix.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static System.DayOfWeek;
+
+namespace HotFix.Test.utilities.scheduling
+{
+    [TestClass]
+    public class get_active
+    {
+        // |  M  |  T  |  W  |  T  |  F  |  S  |  S  |
```

Comment:
Any chance you can append a more detailed explanation along with the beautiful ascii art?